docs/source/migration_guides.md

```diff
@@ -7,7 +7,7 @@ widgets.
 Migrating from 6.0 to 7.0
 -------------------------
 
-For example migrations, see these PRs:
+For example, migrations, see these PRs:
```

Comment:
I think this was okay. It's saying that example migrations are listed below, i.e., it was saying for (example migrations), not (for example) migrations.